common/httpClient.js

```diff
@@ -14,7 +14,6 @@ module.exports = {
         request(
             extend({
                 rejectUnauthorized: false,
-                proxy: config.httpClient.proxy,
```

Comment:
I'd leave the possibility to set the proxy (if null or absent in config it wont make a difference)